packager/media/base/fourccs.h

```diff
@@ -44,6 +44,7 @@ enum FourCC : uint32_t {
   FOURCC_dash = 0x64617368,
   FOURCC_ddts = 0x64647473,
   FOURCC_dec3 = 0x64656333,
+  FOURCC_dac4 = 0x64616334,
```

Comment:
The definition should be sorted. Can you move it after FOURCC_dac3?

Reply:
Sure.